doc_admin/AdminManual.adoc

```diff
@@ -511,8 +514,14 @@ where only a single computer is available. If the facility is large and more cli
 
 include::https://raw.githubusercontent.com/informatici/openhospital/master/oh-bundle/OH-README.md[lines=25..413,leveloffset=+2]
 
+=== 2.3 OH - SERVER mode (portable)
+
+Open Hospital can be lauched in SERVER (portable) mode. SERVER mode allows to launch a local instance of the database server and listen for connections from other clients.
```

Comment:
```suggestion
Open Hospital can be launched in SERVER (portable) mode. SERVER mode launchs a local instance of the database server and listens for connections from other clients.
```

Reply:
Thank you David, fixed, sorry I wrote this quite in a rush for release, it will need some work in the future.